habitat-baselines/habitat_baselines/config/rearrange/rl_hierarchical_oracle_nav.yaml

```diff
@@ -0,0 +1,42 @@
+# @package _global_
+
+# Extends the `rl_hierarchical` config to use an oracle navigation action.
+# Several things change to support the oracle navigation:
+# - Adding the oracle navigation action
+# - Add sliding.
+# - Use the oracle navigation skill.
+
+defaults:
+  - rl_hierarchical
+  - /habitat/task/actions:
```

Comment:
What is the oracle navigation action, and why is it different from the skill?

Reply:
The oracle navigation _action_ is a Habitat-Lab action that accesses the underlying simulator state to compute the oracle path. The oracle navigation _skill_ does not have access to the simulator because it is on the Habitat-Baselines side.